Tests/Unit/MiniAppDownloaderTests.swift

```diff
@@ -0,0 +1,91 @@
+import Quick
+import Nimble
+@testable import MiniApp
+
+// swiftlint:disable function_body_length
+class MiniAppDownloaderTests: QuickSpec {
+
+    override func spec() {
+        describe("mini app folder will be created") {
```

Comment:
Does the SDK code work if the folder already exists? Is that tested?

Reply:
We don't download the file if file exists already for a particular mini app.